src/case/title_case.js

```diff
@@ -11,20 +12,25 @@ import coerceToString from 'helper/string/coerce_to_string';
  * @memberOf Case
  * @param  {string} [subject=''] The string to convert to title case.
  * @param  {Array}  [noSplit]    Do not split words at the specified characters.
+ * @param  {boolean} [preserveUpperCase=false] preserve uppercases.
  * @return {string}              Returns the title case string.
  * @example
  * v.titleCase('learning to fly');
  * // => 'Learning To Fly'
  *
+ * v.titleCase('XMLHttpRequest', preserveUpperCase=true);
```

Comment:
@fatteme Can you change preserveUpperCase=true to simple true? (to make the code runnable)

Reply:
you mean like this?
` v.titleCase('XMLHttpRequest', [], true);`

Reply:
Done